tests/test_georaster.py

```diff
@@ -1403,6 +1397,33 @@ def test_set_nodata(self, example: str) -> None:
             with pytest.raises(ValueError, match=expected_message):
                 # Feed a floating numeric to an integer type
                 r.set_nodata(0.5)
+        elif "float" in r.dtypes[0]:
```

Comment:
Ahah ok, my intention was to use only a single test for both, as the second test also works for uint8 😉

Reply:
Ah yes. Well I guess the test is more precise the current way, as it checks that `integer` types fail with a `nodata` value that would work for `float` types, so I think we can leave as is :)